tests/tt_metal/tt_metal/unit_tests_common/watcher/test_noc_sanitize.cpp

```diff
@@ -139,7 +139,7 @@ void RunTestOnCore(WatcherFixture* fixture, Device* device, CoreCoord &core, boo
             break;
         case SanitizeAlignmentL1:
             expected = fmt::format(
-                "Device {}, {} Core {}[physical {}]: {} using noc0 accesses local L1[addr=0x{:08x},len=102400]",
+                "Device {}, {} Core {}[physical {}]: {} using noc0 accesses local L1[addr=0x{:08x},len=102400] (bad address alignment, reading PCIe/DRAM->L1 requires L1 address to be 32-byte-aligned)",
```

Comment:
I don't think we have a good define to use here, but if so would be better than the "32" in the string

Reply:
Part of #6738 changes, removed from this PR